keyboards/handwired/wulkan/keymaps/default/keymap.c

```diff
@@ -0,0 +1,107 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QWERTY = 0,
```

Comment:
**Optional:**

Just as an FYI, you don't need to assign the first element to `0` explicitly; `enum`s start from a `0` index.

```suggestion
  _QWERTY,
```

Reply:
Yeah that is true, I'm just used to assign the first one for readability but I will remove that one for keeping the standard for this project

Reply:
Oh, it's not a standard, and it's not something that would prevent your pull request from being merged (hence why I said "optional"). If you prefer having it explicitly defined, that's fine.

Reply:
Cool, but i removed it, did see that i did not assign the other enum `unicode_names` so better to at least be persistant :)